EnglishFilter/sections/adservers.txt

```diff
@@ -5,6 +5,7 @@
 !
 !! https://a.bestcontentitem.top/warp/*?r=
 !
+||zopim.com^$third-party
```

Comment:
We have already it in the Annoyances filter.

Reply:
@zloyden Don't make it third party.

Reply:
What do you mean? As I said earlier, it's in the Annoyances filter https://github.com/AdguardTeam/AdguardFilters/blob/master/AnnoyancesFilter/sections/widgets.txt#L176

Reply:
@zloyden Like this: `||zopim.com^$third-party` to this `||zopim.com^` without any domains.

Reply:
Ok wait nevermind, it may cause false positives

Reply:
Yep, that's why we added the domains with `~`. In other words, these are exceptions.

Reply:
Yes.